### PR TITLE
fix: better branch code coverage

### PR DESCRIPTION
**Pull request by viceice**
## Changes
Doing a fix release because some small code changes

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -482,20 +482,24 @@ export const presets: Record<string, Preset> = {};
...
   presets[name] = {
     description: `${name} monorepo`,
-    matchPackagePatterns: is.array(value) ? value : [value],
+    matchPackagePatterns: toArray(value),
   };
 }
+
+function toArray(value: string | string[]): string[] {
+  return is.array(value) ? value : [value];
+}
```
Comment (rarkins) [inline lib/config/presets/internal/monorepo.ts:505]: Can you move this to a generic util file? I bet we'd use it in multiple places

File: lib/modules/datasource/jenkins-plugins/index.spec.ts
Comment (rarkins) [inline lib/modules/datasource/jenkins-plugins/index.spec.ts:1]: Why the changes here?

File: lib/modules/platform/utils/pr-body.ts [hunk 2]
```diff
@@ -16,9 +16,12 @@ export function smartTruncate(input: string, len: number): string {
...
   const divider = `\n\n</details>\n\n---\n\n### Configuration`;
-  const preNotes = reMatch.groups?.preNotes ?? '';
-  const releaseNotes = reMatch.groups?.releaseNotes ?? '';
-  const postNotes = reMatch.groups?.postNotes ?? '';
+  const preNotes =
+    reMatch.groups?.preNotes ?? /* istanbul ignore next: can't happen */ '';
+  const releaseNotes =
+    reMatch.groups?.releaseNotes ?? /* istanbul ignore next: can't happen */ '';
+  const postNotes =
+    reMatch.groups?.postNotes ?? /* istanbul ignore next: can't happen */ '';
```
Comment (rarkins) [inline lib/modules/platform/utils/pr-body.ts:24]: Not `coerceString()` ?

File: lib/modules/versioning/redhat/index.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ class RedhatVersioningApi extends GenericVersioningApi {
 
     const { major, minor, patch, releaseMajor, releaseMinor } = matches;
     const release = [
-      typeof major === 'undefined' ? 0 : Number.parseInt(major, 10),
+      Number.parseInt(major, 10),
```
Comment (rarkins) [inline lib/modules/versioning/redhat/index.ts:23]: major can't be undefined?

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/modules/datasource/jenkins-plugins/index.spec.ts
Comment (viceice) [reply lib/modules/datasource/jenkins-plugins/index.spec.ts:1]: Coverage
https://github.com/renovatebot/renovate/blob/0e81e9df4573a1738bf36979cf20a814261c544f/lib/modules/datasource/jenkins-plugins/index.ts#L75-L78

File: lib/modules/versioning/redhat/index.ts (hunk 3 above)
Comment (viceice) [reply lib/modules/versioning/redhat/index.ts:23]: no, the regex requires that group
https://github.com/renovatebot/renovate/blob/2da8d70d2026b0a300939c14af8d2c5e035f0997/lib/modules/versioning/redhat/index.ts#L10-L12

**Review by rarkins — APPROVED** [review]
